clients/rospy/src/rospy/client.py

```diff
@@ -289,8 +286,7 @@ def init_node(name, argv=None, anonymous=False, log_level=None, disable_rostime=
     # check for name override
     mappings = rospy.names.get_mappings()
     if '__name' in mappings:
-        # use rosgraph version of resolve_name to avoid remapping
-        name = rosgraph.names.resolve_name(mappings['__name'], rospy.core.get_caller_id())
```

Comment:
Just for the review. the second argument (`caller_id=rospy.core.get_caller_id()`) is the default argument and therefore not necessary.